.github/workflows/ci.yml

```diff
@@ -20,7 +19,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version: ['3.9', '3.10', '3.11']
+        python-version: ["3.8", "3.9", "3.10", "3.11"]
```

Comment:
I think we can support 3.8

Reply:
There are open issues with 3.8:
```
FAILED tests/stats/test_intensity_minutes.py::test_weekly_intensity_minutes - AttributeError: 'tuple' object has no attribute 'week'
FAILED tests/stats/test_steps.py::test_weekly_steps - TypeError: unsupported operand type(s) for |: 'dict' and 'dict'
FAILED tests/stats/test_stress.py::test_daily_stress - TypeError: unsupported operand type(s) for |: 'dict' and 'dict'
FAILED tests/stats/test_stress.py::test_daily_stress_pagination - TypeError: unsupported operand type(s) for |: 'dict' and 'dict'
```

We can make the necessary changes to support 3.8, but I recommend doing so outside of this PR.